### DISCUSSION TITLE
Non Export setting

### DISCUSSION THREAD

**Discussion by Bolljohnny** [wlcrs/huawei_solar#782] [Setting up the integration]
Hello,

I can´t find the non export setting when I have integrate the inverter? Is it possible to change the active power to non export via home assistant? I have a smart meter already installed

**Comment by Roving-Ronin** [comment]
If you want to just limit your inverter to ZERO EXPORTS all the time, just run this service in HA Developer Tools: "Huawei Solar: Set Active Power Control to 'Zero power grid connection'"

Can also be set from the Fusion Solar web application, under the sDongle settings.
https://forum.huawei.com/enterprise/en/setting-zero-power-grid-tied-in-the-smart-pv-management-system-smartlogger-scenario/thread/667265450205134848-667213868771979264

Most of this stuff is in the Wiki.... https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-daisy-chained-inverters#combining-multiple-inverter-values

**Reply by Bolljohnny** [reply]
I don’t want it all the time, I just want zero export when it is negative spot price at nordpool. So I want to let the HA fixed it automaticly. Manuelly i can set the zero export via solar web but I want the HA todo this instead. Is it possible?

**Reply by Roving-Ronin** [reply]
Yes use an automation in HA, that uses the Nord power price sensor as a condition (if more than / of less than) and with an action that calls the Huawei service to set max power to x kW and then in reverse step a higher or unlimited law to export when the price changes.

**Reply by Bolljohnny** [reply]
I want to set "the active power setting" to zero export via HA when it´s negative price but I don´t even find this setting in HA now?

**Comment by Bolljohnny** [comment]
I don’t want to set it to a specific kW value because I want my house to
use everything it can. I have a smart meter that knows how much the house
is consuming and how much that it is exporting

lör 3 aug. 2024 kl. 19:22 skrev Roving Ronin ***@***.***>:

**Reply by Roving-Ronin** [reply]
So as per above, use an automation to say:

1. When <name of Export kWh Price sensor> is less than $0.01, then call "Huawei Solar: Limit the power fed to the grid" and set to 0W. 

2. Then conversely another condition that says when <name of Export kWh Price sensor> is more than $0.01 then call "Huawei Solar: Set Active Power Control to Maximum Feed Grid Power" (that will rest inverter to export the max it can produce).

**Reply by Bolljohnny** [reply]
Yes, but how do I do the call? Is it just to write "Huawei Solar: Limit the power fed to the grid" and set to 0W. ?
